### PR TITLE
network: tc: introduce QDiscVTable

### PR DESCRIPTION
**Pull request by yuwata**

### CODE REVIEW

File: src/network/tc/qdisc.c [hunk 1]
```diff
@@ -117,7 +132,6 @@ static int qdisc_handler(sd_netlink *rtnl, sd_netlink_message *m, Link *link) {
 int qdisc_configure(Link *link, QDisc *qdisc) {
         _cleanup_(sd_netlink_message_unrefp) sd_netlink_message *req = NULL;
         _cleanup_free_ char *tca_kind = NULL;
```
Comment (mrc0mmand) [inline src/network/tc/qdisc.c]:
```
    archlinux_systemd_ci: ../src/network/tc/qdisc.c:134:30: error: unused variable 'tca_kind' [-Werror,-Wunused-variable]
    archlinux_systemd_ci:         _cleanup_free_ char *tca_kind = NULL;
    archlinux_systemd_ci:                              ^
    archlinux_systemd_ci: 1 error generated.
```

Comment (yuwata) [reply src/network/tc/qdisc.c]: Thanks. Fixed.

**Comment by ssahani** [comment]
vtables always nice. CI passed. LGTM. thanks.

**Comment by yuwata** [comment]
@ssahani Thank you for the review. Merging.

**Pull request merged**